fleetd/fleet.go

```diff
@@ -213,6 +222,33 @@ func listenForSignals(sigmap map[os.Signal]func()) {
 	}
 }
 
+func createPidFile(pidfile string) error {
+	if pidstring, err := ioutil.ReadFile(pidfile); err == nil {
+		pid, err := strconv.Atoi(string(pidstring))
+		if err == nil {
+			if _, err := os.Stat(fmt.Sprintf("/proc/%d/", pid)); err == nil {
```

Comment:
This stale pidfile check can be spoofed if some other process now has this pid. Which makes me question the whole premise of this change: Isn't the locking provided by opening network sockets good enough to prevent multiple fleet daemons from running?